manifests/profiles/external.yaml

```diff
@@ -19,4 +19,5 @@ spec:
     pilot:
       configMap: false
     global:
+      externalIstiod: true
```

Comment:
Added the flag here so that the ClusterRole includes the extra permissions on config clusters when the `istioctl install` is run on the config cluster AFTER the `istioctl x create_remote_secret`, since in that order it overwrites the ClusterRole that was created with the extra permissions.

Strange, however, that they don't seem to really be needed anyway. The external istiod has no trouble creating the leader-election lock on the config cluster, with or without the added permission. Only on other remote clusters does it seem to be needed. @stevenctl any idea why?

Reply:
I think the config cluster is accessed via the `istiod` service account instead of the `istio-reader` sa

Reply:
Make sense. This is another good reason to make it optional in create_remote_secret in a future PR.